### PR TITLE
Drop support for legacy Python 2

### PR DESCRIPTION
**Pull request by hugovk**
Fixes https://github.com/omnisci/pymapd/issues/131.

### CODE REVIEW

**Comment by randyzwitch** [comment]
Thanks! I’ll review this Monday when I’m back at work

File: mapd/MapD-remote [hunk 1]
```diff
@@ -9,10 +9,7 @@
 
 import sys
 import pprint
-if sys.version_info[0] > 2:
```
Comment (randyzwitch) [inline mapd/MapD-remote]: Although the code in the `mapd` folders technically supports Python 2, please remove all of the changes from the `mapd` folder from this PR. This code is auto-generated by Apache Thrift compiler, so if we merge these changes in, the next time we update the Thrift code the diff will be too large to evaluate

File: mapd/MapD.py [hunk 2]
```diff
@@ -9,15 +9,14 @@
 from thrift.Thrift import TType, TMessageType, TApplicationException
```
Comment (randyzwitch) [inline mapd/MapD.py]: Remove all of the changes on this file

File: mapd/ttypes.py [hunk 3]
```diff
@@ -9,13 +9,11 @@
 from thrift.Thrift import TType, TException, TApplicationException
```
Comment (randyzwitch) [inline mapd/ttypes.py]: Please remove all the changes to this file as well

File: mapd/MapD-remote (hunk 1 above)
Comment (hugovk) [reply mapd/MapD-remote]: Done!

**Pull request merged**